modules/tests/src/test/scala/ch/epfl/scala/debugadapter/internal/RuntimeEvaluatorTests.scala

```diff
@@ -963,6 +963,56 @@ abstract class RuntimeEvaluatorTests(val scalaVersion: ScalaVersion) extends Deb
       Evaluation.success("bar.Baz(42).Bizz.x", 43)
     )
   }
+
+  test("should assign values to local var and fields") {
+    val source =
+      """|package example
+         |
+         |class A {
+         |  var a = 0
+         |  var l = new B
+         |  def f = this
+         |}
+         |
+         |class B
+         |class C extends B {
+         |  var c = 41
+         |}
+         |
+         |object Main {
+         |  var b = 0
+         |  var c = 0 
+         |  def main(args: Array[String]): Unit = {
+         |    val a = new A
+         |    var b = 0
+         |    var bc: B = new C
+         |    println("ok")
+         |  }
+         |}
+         |""".stripMargin
+    implicit val debuggee: TestingDebuggee = TestingDebuggee.mainClass(source, "example.Main", scalaVersion)
+    def assignSuccess(expr: String) = Evaluation.success(expr)(res => res == "<void value>")
```

Comment:
Would it be possible to write it this way?

```suggestion
    def assignSuccess(expr: String) = Evaluation.success(expr, ())
```

Reply:
Nop, because I return a `mirrorOfVoid` after the assignment